app/controllers/charges_controller.rb

```diff
@@ -34,6 +34,13 @@ def destroy
       flash[:notice] = "\"#{current_user.email}\" was downgraded to standard successfully."
       current_user.role = 'standard'
       current_user.save!
+      wikis = current_user.wikis
```

Comment:
Very nice handling of this situation. A good refactor would be to put lines 35-43 into the `User` model in a `downgrade_user` method and then just call the method in the controller, keeping our [controller skinny](http://weblog.jamisbuck.org/2006/10/18/skinny-controller-fat-model)